Add Windows Guide (#1380)
privacyguides/privacyguides.org: docs/windows/overview.md

---
title: Windows Overview
icon: material/microsoft-windows
---

## Windows

Windows is a proprietary operating system created by Microsoft Inc. in 1985. It is primarily focused on personal computing and is now the most popular desktop OS, used by about [75%](https://gs.statcounter.com/os-market-share/desktop/worldwide) of all desktop users. However, it has own privacy and security issues.

## Issues present in Windows

> Over the years, Microsoft has demonstrated a lot of privacy-invasive behaviour with their software and services. They have continually taken advantage of the fact that Windows is the most widely-used desktop OS, and that most people don't change the default settings, in order to collect users' personal information.

Reviewer: ```suggestion
Over the years, Microsoft has demonstrated a lot of privacy-invasive behaviour with their software and services. They have continually taken advantage of the fact that Windows is the most wide-used desktop OS, and that most people don't change the default settings, in order to collect users' personal information.
```

Windows 10 was [criticised](https://www.theguardian.com/technology/2015/jul/31/windows-10-microsoft-faces-criticism-over-privacy-default-settings) for having default settings that sent a lot of data and telemetry back to Microsoft, including:

!!! quote "[Criticism of Microsoft - Wikipedia](https://en.wikipedia.org/wiki/Criticism_of_Microsoft#Telemetry_and_data_collection)"
    user contacts and calendar events, location data and history, "telemetry" (diagnostics data) ... an "advertising ID", as well as further data when the Cortana assistant is enabled in full

> At launch, telemetry could not be disabled in non-enterprise editions of Windows 10. Only after [criticism](https://www.theverge.com/2016/7/21/12246266/france-microsoft-privacy-windows-10-cnil) from the France data protection commision, the [Electronic Frontier Foundation](https://www.eff.org/deeplinks/2016/08/windows-10-microsoft-blatantly-disregards-user-choice-and-privacy-deep-dive) and the [European Union](https://www.reuters.com/article/us-microsoft-dataprotection-eu-idUSKBN15Z1UI), Microsoft changed the way they collect telemetry, allowing users to choose between "Basic" and "Full", with "Basic" mode collecting [much less telemetry](https://www.extremetech.com/computing/243079-upcoming-windows-update-reduces-spying-microsoft-still-mum-data-collects). Along with that, Microsoft collects a [lot more data from Windows 10](https://web.archive.org/web/20210711143017/https://privacytools.io/operating-systems/#win10).

Reviewer: ```suggestion
At launch, telemetry could not be disabled in non-enterprise editions of Windows 10. Only after [criticism](https://www.theverge.com/2016/7/21/12246266/france-microsoft-privacy-windows-10-cnil) from the France data protection commission, the [Electronic Frontier Foundation](https://www.eff.org/deeplinks/2016/08/windows-10-microsoft-blatantly-disregards-user-choice-and-privacy-deep-dive) and the [European Union](https://www.reuters.com/article/us-microsoft-dataprotection-eu-idUSKBN15Z1UI), Microsoft changed the way they collect telemetry, allowing users to choose between "Basic" and "Full", with "Basic" mode collecting [much less telemetry](https://www.extremetech.com/computing/243079-upcoming-windows-update-reduces-spying-microsoft-still-mum-data-collects). Along with that, Microsoft collects a [lot more data from Windows 10](https://web.archive.org/web/20210711143017/https://privacytools.io/operating-systems/#win10).
```

With the launch of Windows 11, a lot of [other](https://www.windowscentral.com/one-thing-microsoft-didnt-discuss-windows-11-privacy) [concerns](https://www.pcworld.com/article/539183/windows-11-review-an-unnecessary-replacement-for-windows-10.html) were raised, such as:

- integration of Microsoft Teams into the OS, which would encourage users to switch to the service, allowing Microsoft to collect even more data.
- making it difficult to switch away from Edge by having to set a default app for each file type, and not allowing the `Make default` buttons in other browsers to function properly.
- removing the ability to have local accounts in Windows 11 Home, therefore forcing you to log into a Microsoft account.
- having all data collection options on by default
- working with Amazon to bring Android apps to Windows through the Windows Subsystem for Android, likely allowing both Microsoft and Amazon to collect data about Android app usage on Windows.
- Using users in a P2P way to distribute Windows updates to reduce load in Microsoft's servers without users' consent.

> ## Choosing your Edition of Windows

Reviewer: ```suggestion
## Choosing your Windows edition
```

> While using Windows it is better to select either Windows **Enterprise** Edition or **Education** Edition because it gives more control over hardening it for privacy and security by giving refined access such as disabling full diagnostic data using GP Editor.

Reviewer: ```suggestion
While using Windows, it is better to select either Windows **Enterprise** Edition or **Education** Edition because it gives more control over hardening it for privacy and security by giving refined access such as disabling full diagnostic data using GP Editor.
```

> If you cannot get the above you must opt for **Professional** Edition.

Reviewer: ```suggestion
If you cannot get what's above, you must opt for **Professional** Edition.
```

#### Editions to avoid 

- It is not recommended to use forks or modified versions of Windows such as Windows AME. It should be avoided at all cost. Since modified versions of Windows, such as AME, don't get updates, antivirus programs like Defender can fall out of date or be disabled entirely, opening you up to attacks.

> - Windows **Home** edition is **not** recommended as it do not have many advantages that Professional edition provide such as Bitlocker Drive Encryption, Hyper-V, Windows Sandbox, etc it also uploads Bitlocker Encryption keys to Microsoft servers which actually defies the aspect of encryting implemented in a different way.

Reviewer: ```suggestion
- Windows **Home** edition is **not** recommended as it does not have many advantages that Professional edition provides such as Bitlocker Drive Encryption, Hyper-V, Windows Sandbox, etc. It also uploads Bitlocker Encryption keys to Microsoft servers which actually defies the aspect of encrypting implemented in a different way.
```

##### Recommendations

> We recommend you choose Windows 11 over Windows 10 as it is the latest version, and brings many security-related improvements with it by default such as [Secure Boot](https://docs.microsoft.com/en-us/windows-hardware/design/device-experiences/oem-secure-boot), [VBS](https://docs.microsoft.com/en-us/windows-hardware/design/device-experiences/oem-vbs), [HVCI](https://docs.microsoft.com/en-us/windows-hardware/drivers/bringup/device-guard-and-credential-guard), etc. Windows 10 will stop getting updates after [October 14, 2025](https://docs.microsoft.com/en-us/lifecycle/products/windows-10-home-and-pro).

Reviewer: ```suggestion
We recommend you choose Windows 11 over Windows 10 as it is the latest version and brings many security-related improvements with it by default such as [Secure Boot](https://docs.microsoft.com/en-us/windows-hardware/design/device-experiences/oem-secure-boot), [VBS](https://docs.microsoft.com/en-us/windows-hardware/design/device-experiences/oem-vbs), [HVCI](https://docs.microsoft.com/en-us/windows-hardware/drivers/bringup/device-guard-and-credential-guard), etc. Windows 10 will stop getting updates after [October 14, 2025](https://docs.microsoft.com/en-us/lifecycle/products/windows-10-home-and-pro).
```

### Installing Windows

We recommend that you use the official [Media Creation tool](https://www.microsoft.com/software-download/windows11) to flash the ISO to the USB, over third-party options such as Rufus, Balena Etcher, etc., so that you don't tamper the ISO.

> You should opt-for Multi edition ISO so that you can install Education or Enterprise edition.

Reviewer: ```suggestion
You should opt for multi-edition ISO so that you can install Education or Enterprise edition.
```

- removing the ability to have local accounts in Windows 11 Home, therefore forcing you to log into a Microsoft account.
- having all data collection options on by default
- working with Amazon to bring Android apps to Windows through the Windows Subsystem for Android, likely allowing both Microsoft and Amazon to collect data about Android app usage on Windows.
- Using users in a P2P way to distribute Windows updates to reduce load in Microsoft's servers without users' consent.

## Choosing your Edition of Windows

> While using Windows it is better to select either Windows **Enterprise** Edition or **Education** Edition because it gives more control over hardening it for privacy and security by giving refined access such as disabling full Diagnostic data using GP Editor.

Reviewer 2: `Diagnostic` -> `diagnostic`

- working with Amazon to bring Android apps to Windows through the Windows Subsystem for Android, likely allowing both Microsoft and Amazon to collect data about Android app usage on Windows.
- Using users in a Peer-to-Peer way to distribute Windows updates to reduce load in Microsoft's Servers without users Consent.

## Choosing your Edition of Windows

While using Windows it is better to select either Windows **Enterprise** Edition or **Education** Edition because it gives more control over hardening it for privacy and security by giving access to more Group Policy Objects in the GP Editor.

If you cannot get the above you must opt for **Professional** Edition because Home edition uploads Bitlocker Encryption keys to Microsoft servers which actually defies the aspect of encrypting.

It is not recommended to use forks or modified versions of Windows such as Windows AME. It should be avoided at all cost. Because versions such as AME do not get updates, No AVs such as Defender is disabled opening yourself to attacks and other critical Components too. It is only AME when you use the device fully offline and not connecting to the Internet.

> We recommend you choose Windows 11 over Windows 10 as it is the latest version, and brings many security-related improvements with it by default such as Secure Boot, VBS, HVCI, etc. Windows 10 will stop getting updates after [October 14, 2025](https://docs.microsoft.com/en-us/lifecycle/products/windows-10-home-and-pro).

Author: I have added Abbreviation, Admonitions at last. Perhpas, It would be right. I think.

> We recommend you choose Windows 11 over Windows 10 as it is the latest version, and brings many security-related improvements with it by default such as Secure Boot, VBS, HVCI, etc. Windows 10 will stop getting updates after [October 14, 2025](https://docs.microsoft.com/en-us/lifecycle/products/windows-10-home-and-pro).

Reviewer 2: I was suggesting it because it would save readers having to manually search for it and potentially finding incorrect information. If we link to the documentation directly then there's no risk of that.

> We recommend you choose Windows 11 over Windows 10 as it is the latest version, and brings many security-related improvements with it by default such as Secure Boot, VBS, HVCI, etc. Windows 10 will stop getting updates after [October 14, 2025](https://docs.microsoft.com/en-us/lifecycle/products/windows-10-home-and-pro).

Author: I could do that. But the Admonitions doesn't work which I added at last in my local build. What do you think ?
